pyproject.toml

```diff
@@ -62,6 +62,11 @@ asyncio_mode = "auto"
 [tool.coverage.report]
 exclude_also = ["if typing.TYPE_CHECKING:"]
 
+[tool.coverage.run]
+omit = [
+    "*/tests/*"
+]
```

Comment:
why omitting tests from coverage? tests are also code and coverage helps to find unused sections in tests

Reply:
I completely agree that tests are also code, but I thought the folder with tests by default is not supposed to check coverage. but of course I will do as you say

Reply:
for me, coverage for tests is very useful, thank you for understanding